pkg/executables/flux_test.go

```diff
@@ -433,3 +441,109 @@ func TestFluxReconcile(t *testing.T) {
 		})
 	}
 }
+
+func TestFluxInstallGitToolkitsSuccess(t *testing.T) {
```

Comment:
can we name this based on the tested method?
```suggestion
func TestFluxBootstrapToolkitsComponentsGitSuccess(t *testing.T) {
```